src/lib/components/event/event-summary-row.svelte

```diff
@@ -1,13 +1,25 @@
 <script lang="ts">
+  import { page } from '$app/stores';
+
+  import { getGroupForEvent, isEventGroup } from '$lib/models/group-events';
+
   import { formatDate } from '$lib/utilities/format-date';
 
   import EventDetails from './event-details.svelte';
-  import EventSingleDetail from './event-single-detail.svelte';
+  import EventGroupDetails from './event-group-details.svelte';
+
+  export let event: IterableEvent;
+  export let expandAll = false;
+  export let compact = false;
+
+  let selectedId = event.id;
 
-  export let event: HistoryEventWithId;
-  export let expandAll: boolean = false;
+  let eventGroup = isEventGroup(event)
+    ? event
+    : getGroupForEvent(event, $page.stuff.eventGroups);
```

Comment:
Gonna need to do getGroupForEvent(event, $page?.stuff?.eventGroups ?? $importEventGroups) to make this refactor work with the importer since the event groups are set in a store and not on the load function (stuff)

Reply:
Ah, maybe we should just use context or pass it down instead.

Reply:
Yeah pass it on down to the component.